sdk/response.js

```diff
@@ -84,10 +84,7 @@ class Response extends Api {
    * @return {boolean}
    */
   hasReturn() {
-    return this
-      .getServiceSchema(this._name, this._version)
-      .getActionSchema(this._actionName)
-      .hasReturn();
+    return this.return !== undefined;
```

Comment:
Where is the `return` property previously set? It doesn't seem to be declared on either the `Api` class or the `Response` class.